src/main/java/frc/robot/RobotContainer.java

```diff
@@ -49,9 +50,9 @@
     // private final DriveTrain driveTrain = new DriveTrain();
     // private final UltrasonicSensor ultrasonicSensor = new UltrasonicSensor();
 
-    private final Climber climber = new Climber();
-    private final Intake intake = new Intake();
-
+    public static final Climber climber = new Climber();
```

Comment:
Move all of these into Robot.java, check the clean up PR. that is the right way to do it.

Reply:
We can do this when it's merged